src/scopes/jsObjects.ts

```diff
@@ -2,6 +2,10 @@ import type { Infer } from "../main.ts"
 import type { inferObjectKind } from "../utils/objectKinds.ts"
 import { scope } from "./scope.ts"
 
+/**
+ * @scope
+ * @descriptions descriptions: { "Object": "an object","Array": "an array","Function": "a function", "Date": "a Date","RegExp": "a RegExp","Error": "an Error","Map": "a Map","Set": "a Set","String": "a String object","Number": "a Number object","Boolean": "a Boolean object","Promise": "a Promise","WeakMap": "a WeakMap","WeakSet": "a WeakSet" }
```

Comment:
Can we remove these descriptions? All the JSObjects are self-explanatory. Also, let's change "descriptions" to keywords since I think the implementation was kind of scope specific?